nemo/src/io/parser.rs

```diff
@@ -326,6 +327,30 @@ impl<'a> RuleParser<'a> {
         traced("parse_close_parenthesis", space_delimited_token(")"))
     }
 
+    /// Matches an opening parenthesis,
+    /// then gets an object from the parser,
+    /// and finally matches an closing parenthesis.
+    pub fn in_parenthesis<'b, O, F>(
```

Comment:
Maybe rename this to `parenthesised`, which fits in better with the other combinators? Otherwise needs to at least be pluralised, so `in_parentheses`.